website/docs/r/globalaccelerator_endpoint_group.html.markdown

```diff
@@ -30,7 +30,7 @@ The following arguments are supported:
 * `listener_arn` - (Required) The Amazon Resource Name (ARN) of the listener.
 * `endpoint_group_region` (Optional) - The name of the AWS Region where the endpoint group is located.
 * `health_check_interval_seconds` - (Optional) The time—10 seconds or 30 seconds—between each health check for an endpoint. The default value is 30.
-* `health_check_path` - (Optional) If the protocol is HTTP/S, then this specifies the path that is the destination for health check targets. The default value is slash (/).
+* `health_check_path` - (Optional) If the protocol is HTTP/S, then this specifies the path that is the destination for health check targets.
```

Comment:
Let's add back `/` as the default for HTTP(S) (as it is 😄) and add a note about how changing to `Computed` affects drift detection.

```suggestion
* `health_check_path` - (Optional) If the protocol is HTTP/S, then this specifies the path that is the destination for health check targets. The default value is slash (`/`). Terraform will only perform drift detection of its value when present in a configuration.
```

Reply:
Hrm, but we don't have `Default: "/"` now? So it really depends on what AWS side decide to put as default. Or at least limit that to "The default value for HTTP/HTTPS ..."

Agree on the drift detection part.

Reply:
I verified that AWS do return `/` if nothing is specified on creation (for HTTP(S) health check protocol).

Reply:
You can decide on the wording 😄.

Reply:
@ewbankkit Given the sentence started with "If the protocol is HTTP/S" I felt yours read just fine :D

I was thinking the drift detection part might be reword like this "Terraform will only perform drift detection for this value if changed by the user.", but I'm not sure that's entirely correct (?)

Reply:
I don't think we need to mention `TCP` as that's kind of implicit from not `HTTP/S`.
There seems to be no consistency in the existing documentation over the drift detection wording; I see

* `A configuration must be present for this argument to perform drift detection`
* `Terraform will only perform drift detection of its value when present in a configuration`
* `Terraform will only perform drift detection if a configuration value is provided`
* `Must be configured to perform drift detection`
* `To enable drift detection, must be configured`

So I suggest picking one of the above to avoid a 6th 😄.

Reply:
Totally agree.

Ehm.......... I thought I've incorporated your suggested changes.... let me check what happened to my `git` :facepalm: